ssh.go

```diff
@@ -875,16 +876,7 @@ func sshPrintTunnel(ifce *Interface, fs interface{}, a []string, w sshd.StringWr
 	return enc.Encode(copyHostInfo(hostInfo, ifce.hostMap.preferredRanges))
 }
 
-func sshReload(fs interface{}, a []string, w sshd.StringWriter) error {
-	p, err := os.FindProcess(os.Getpid())
-	if err != nil {
-		return w.WriteLine(err.Error())
-		//TODO
-	}
-	err = p.Signal(syscall.SIGHUP)
-	if err != nil {
-		return w.WriteLine(err.Error())
-		//TODO
-	}
-	return w.WriteLine("HUP sent")
+func sshReload(c *config.C, w sshd.StringWriter) error {
+    c.ReloadConfig()
+	return w.WriteLine("Config reloaded")
```

Comment:
Per the PR description, IIUC this will never be seen because `c.ReloadConfig()` is a blocking call that restarts the SSH server.

Even if we change that line to `go c.ReloadConfig()` we'll be racing it. I agree then that this message should be sent prior to `c.ReloadConfig()`, e.g. "Reloading config"